### PR TITLE
Create sap_redwood_traversal.md

### PR DESCRIPTION
**Pull request by vah13**
I wrote some docs, yo.

### CODE REVIEW

**Comment by bcoles** [comment]
Delayed pending #10353

File: documentation/modules/auxiliary/scanner/sap/sap_redwood_traversal.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Description
+This is a module for exploitation an anon directory traversal vulnerability in SAP NetWeaver Application Server Java in Redwood module. The vulnerability was active exploited from Chines hackers in the wild in 2017.
```
Comment (bcoles) [inline documentation/modules/auxiliary/scanner/sap/sap_redwood_traversal.md]: Fixed some typos/grammar:

```
This is a module for exploitation of an unauthenticated directory traversal vulnerability in SAP NetWeaver Application Server Java in Redwood module. The vulnerability was active exploited by Chinese hackers in the wild in 2017.
```

File: documentation/modules/auxiliary/scanner/sap/sap_redwood_traversal.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+
+# Verification Steps
+```
+Start msfconsole
```
Comment (bcoles) [inline documentation/modules/auxiliary/scanner/sap/sap_redwood_traversal.md]: This should be a numbered list.

Also, real examples are nice, ie `set FILEPATH /etc/passwd`.

```
1. Start msfconsole
2. use auxiliary/scanner/sap/sap_redwood_traversal
3. set RHOSTS [IP]
4. set FILEPATH /etc/passwd
5. set DEPTH 15
6. run
```

**Comment by busterb** [comment]
I cherry picked the docs into PR #10353

**Pull request closed without merging**